### DISCUSSION TITLE
Wont load past '[ok] Reached target Path Units.'

### DISCUSSION THREAD

**Discussion by OSSDTech** [Security-Onion-Solutions/securityonion#12139] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

16

### Storage for /

500

### Storage for /nsm

500

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am installing a new install on a VM and it won't boot past '[ ok ] Reached target Path Units.'  I have tried creating multiple VMs and have downloaded multiple copies of the 2.4.30 Iso.  All end up stuck at this step.  If I try an Iso from previous version it works without issue.  Any thoughts on what would cause this to stop the setup/boot?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by cm-ops** [comment]
Have you verified the ISO? https://docs.securityonion.net/en/2.4/download.html

**Reply by OSSDTech** [reply]
Yes,  on both copies that I downloaded and everything matched.

**Comment by OSSDTech** [comment]
resolved

**Reply by reyesj2** [reply]
Was this due to the VM cpu type not being set to "host"? 

If you can please share what the resolution was so others encountering this issue can find the answer.

**Comment by pimpernoni** [comment]
> resolved

Can you share how you resolved this. I am doing the same setup on ProxMox prior to Launching SO 2.4 in prod. I can't seem to figure out why it freezes.
Thanks
P
